worker/include/Channel/UnixStreamSocket.hpp

```diff
@@ -38,6 +38,19 @@ namespace Channel
 		// Others.
 		size_t msgStart{ 0 }; // Where the latest message starts.
 	};
+
+	class ChannelWrapper
+	{
+	public:
+		explicit ChannelWrapper(int consumerFd, int producerFd)
+		  : consumerSocket(consumerFd, ::UnixStreamSocket::SocketRole::CONSUMER),
+		    producerSocket(producerFd, ::UnixStreamSocket::SocketRole::PRODUCER)
+		{
+		}
+
```

Comment:
Let's add a `public` line here to separate methods and members.

Reply:
yep. already added.